### DISCUSSION TITLE
昨天发现，cloudflare国际官网强制跳转到了中国官网了

### DISCUSSION THREAD

**Discussion by uloveme** [XIU2/CloudflareSpeedTest#299] [分享]
更神奇的是，今天居然解除限速了。
我电信的只能跑满只有10兆，早上都只有这个速度，但是十点以后就可以到50兆了（梯子限速60），看4K没问题了

**Comment by XIU2** [comment]
老黄历了，Cloudflare 中国版官网上线有一段时间了。

另外，这种和 CloudflareST 本身无关的就不要发 Issues 了（这里是用来反馈问题的）。
去 https://github.com/XIU2/CloudflareSpeedTest/discussions （讨论区）发更合适一些。
